src/Microsoft.NetCore.Analyzers/Core/Security/UseDefaultDllImportSearchPathsAttribute.cs

```diff
@@ -21,15 +21,15 @@ public sealed class UseDefaultDllImportSearchPathsAttribute : DiagnosticAnalyzer
             typeof(MicrosoftNetCoreAnalyzersResources),
             nameof(MicrosoftNetCoreAnalyzersResources.UseDefaultDllImportSearchPathsAttribute),
             nameof(MicrosoftNetCoreAnalyzersResources.UseDefaultDllImportSearchPathsAttributeMessage),
-            DiagnosticHelpers.EnabledByDefaultIfNotBuildingVSIX,
+            false,
```

Comment:
Would be good to even update the documentation files, though that can come in a separate PR just prior to the release.

Reply:
One day I'll remember.  Today is not that day.  🙂

Reply:
We should just fix the tooling to auto-generate these PRs instead!